packages/angular/projects/clr-angular/src/timeline/interface/icon-attribute.iterface.ts

```diff
@@ -4,7 +4,10 @@
  * The full license information can be found in LICENSE in the root directory of this project.
```

Comment:
Filename is `icon-attribute.iterface.ts` instead of `icon-attribute.interface.ts`